apps/cwl_firehose_subscriber/Makefile

```diff
@@ -13,15 +23,17 @@ test:
 build:
```

Comment:
build not dependent on install? how does venv get initialized then?

Reply:
I was just calling it in a separate step in the .gitlab-yml